### DISCUSSION TITLE
Also included Italian websites?

### DISCUSSION THREAD

**Discussion by openaspace** [sefinek24/Sefinek-Blocklist-Collection#60] [General]
Hello, Italian fake news websites are included?

**Comment by sefinek** [comment]
Probably: https://github.com/search?q=repo%3Asefinek%2FSefinek-Blocklist-Collection+.it

If you have any blocklists for Italy, feel free to share them <:

**Reply by openaspace** [reply]
Thank you!

I'm searching for Italian fake news filters for elderly people..

Exist also the way to filter the Facebook posts? 😅🥴

**Reply by sefinek** [reply]
Pi-hole or AdGuard only operate at the network level, which means they can't filter content like Facebook posts. DNS servers are only capable of blocking specific domains. However, AdGuard is better suited for tasks like parental control, as it offers features such as Safe Search. I can't provide much more information because I haven't used AdGuard extensively. You can learn more here: https://adguard.com/en/blog/adguard-parental-control.html

Out of curiosity, I tried searching for dedicated lists for Italy, but honestly, I couldn't find any
